pkg/security/security_profile/dump/remote_storage.go

```diff
@@ -69,7 +69,9 @@ func NewActivityDumpRemoteStorage() (ActivityDumpStorage, error) {
 	}
 	for _, endpoint := range endpoints.GetReliableEndpoints() {
 		storage.urls = append(storage.urls, utils.GetEndpointURL(endpoint, "api/v2/secdump"))
-		storage.apiKeys = append(storage.apiKeys, endpoint.APIKey)
+		// TODO - runtime API key refresh: Storing the API key like this will no longer be valid once the
```

Comment:
is it fine to keep like that for now ? Do you have documentation on what we do to onboard into this ?

Reply:
The feature is still in dev, so we don't have an official documentation yet. We did a presentation to the agent space which was recorder, I'll send you the link.